keepassxc-browser/background/page.js

```diff
@@ -372,9 +377,15 @@ const createContextMenuItem = function({action, args, ...options}) {
                 action: action,
                 args: args
             }).catch((err) => {
-                console.log(err);
+                showErrorMessage(err);
             });
         },
         ...options
     });
 };
+
+const debugLog = function(message, extra) {
```

Comment:
I think this function redirection is not worthwhile. Can we just have showDebugMessage and check for the setting being enabled in there?

Reply:
The problem is that `page.settings` is not directly available for content scripts, unless we send a message to background script every time we write a debug message and check it. This is why content scripts use `kpxc.settings` for checking because it's already parsed on page load. And, vice versa, `kpxc.settings` is not directly available background scripts.

Reply:
Damn I figured that is why. It's just awkward to have two very different function names for logging errors vs debug.

Reply:
Maybe those function names could be switched after all. The internal call should be `debugLog()` and the ones we call are `showErrorMessage()` and `showDebugMessage()`.

Reply:
show or log? I prefer log cause that's what you are doing. logError, logDebug

Reply:
Let's use `log`.